api_app/api/routes/workspace_templates.py

```diff
@@ -22,7 +22,7 @@ def get_current_template_by_name(template_name: str, template_repo: ResourceTemp
         template = template_repo.get_current_template(template_name, resource_type, parent_service_template_name)
         return template_repo.enrich_template(template)
     except EntityDoesNotExist:
-        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=strings.TEMPLATE_DOES_NOT_EXIST)
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=strings.WORKSPACE_SERVICE_TEMPLATE_DOES_NOT_EXIST)
```

Comment:
I believe this one needs to remain as `resource_type: ResourceType` might be of a different type.

Reply:
okay, so both error messages(`TEMPLATE_DOES_NOT_EXIST` and `WORKSPACE_SERVICE_TEMPLATE_DOES_NOT_EXIST`) will be there. right?

Reply:
In this place just `TEMPLATE_DOES_NOT_EXIST` as we do not know the resourceType. Ideally the `resourceType` passed in would be substituted into the error but happy with either for now.